src/acceptance/api/api_test.go

```diff
@@ -182,11 +182,6 @@ var _ = Describe("AutoScaler Public API", func() {
 
 			BeforeEach(func() {
 				totalTime := time.Duration(cfg.AggregateInterval*2)*time.Second + 3*time.Minute
-
-				Eventually(func() uint64 {
```

Comment:
this is a pointless assert .... well just to intimately entwined with the running of the autoscaler.
All we care about is did it scale.